src/main/java/org/graylog/integrations/ipfix/codecs/IpfixCodec.java

```diff
@@ -82,18 +91,54 @@
     protected IpfixCodec(@Assisted Configuration configuration, IpfixAggregator ipfixAggregator) throws IOException {
         super(configuration);
         this.ipfixAggregator = ipfixAggregator;
-        // Standard IPFIX definition file is loaded from the classpath and is mandatory. We don't check for null or empty.
+
         final URL standardIPFixDefTemplate = Resources.getResource(IpfixCodec.class, IPFIX_STANDARD_DEFINITION);
-        final String ipFixCustomDefPath = (String) configuration.getSource().get(CK_IPFIX_DEFINITION_PATH);
-        if (ipFixCustomDefPath == null || ipFixCustomDefPath.trim().isEmpty()) {
+        final List<String> customDefFilePathList = configuration.getList(CK_IPFIX_DEFINITION_PATH);
+        List<URL> filePaths = new ArrayList<>();
+
+        if (customDefFilePathList == null || customDefFilePathList.isEmpty()) {
```

Comment:
@bernd @danotorrey - Below are some of my thoughts and wanted to share to bounce some ideas with you.
`Configuration.getURL(CK_IPFIX_DEFINITION_PATH)` can be a good  addition to the core code , that returns a 
`List<URL>`
L99-L108 can be refactored, with a code like below and would require us to slow think about this design.
```
ieds = Builder.build(InformationElementDefinitions.class)
                              .standardDef(standardIPFixDefTemplate)
                              .customDef(customDefFilePathList)
```